src/olympia/editors/tests/test_views.py

```diff
@@ -760,7 +763,7 @@ def _test_results(self):
             assert latest_version
             name = '%s %s' % (unicode(addon.name),
                               latest_version.version)
-            channel = ['unlisted'] if not self.listed else []
+            channel = [self.channel_name] if self.channel_name else []
```

Comment:
hmm.... I don't like this change 😞 
Can we have channel_name as a sensible default (`listed`?) - `None` is never a valid channel - and test accordingly?

Reply:
Something like this?

```diff
diff --git a/src/olympia/editors/tests/test_views.py b/src/olympia/editors/tests/test_views.py
index 4b5cb6d..63f618d 100644
--- a/src/olympia/editors/tests/test_views.py
+++ b/src/olympia/editors/tests/test_views.py
@@ -661,7 +661,7 @@ class QueueTest(EditorTest):
         self.url = reverse('editors.queue_pending')
         self.addons = OrderedDict()
         self.expected_addons = []
-        self.channel_name = 'unlisted' if not self.listed else None
+        self.channel_name = 'listed' if self.listed else 'unlisted'
 
     def generate_files(self, subset=None, files=None):
         if subset is None:
@@ -763,7 +763,12 @@ class QueueTest(EditorTest):
             assert latest_version
             name = '%s %s' % (unicode(addon.name),
                               latest_version.version)
-            channel = [self.channel_name] if self.channel_name else []
+            if self.channel_name == 'listed':
+                # We typically don't include the channel name if it's the
+                # default one, 'listed'.
+                channel = []
+            else:
+                channel = [self.channel_name]
             url = reverse('editors.review', args=channel + [addon.slug])
             expected.append((name, url))
         links = pq(

```

Reply:
👍

Reply:
Done in 437e3427ce5269470eda1fd68b21582b4cc4ca66